### ISSUE TITLE
mount flag to run vfs/refresh recursive=true

### ISSUE THREAD

**Issue by beyondmeat** [rclone/rclone#6830]
#### The associated forum post URL from `https://forum.rclone.org`
n/a

#### What is your current rclone version (output from `rclone version`)?
1.6.1

#### What problem are you are trying to solve?
vfs/refresh w/o needing --rc

#### How do you think rclone should be changed to solve that?
add a flag to be used on mounts that would warm up the vfs cache.
I don't want to enable --rc because I don't need to except to do this at service startup.

A flag could do this.

#### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

**Comment by ncw** [comment]
Does rclone need to do this before the mount is ready or can it do it in the background after the mount is declared ready?

**Comment by beyondmeat** [comment]
I would say async / in the background. I see no point of delaying the mount for it.

**Comment by ncw** [comment]
I think this would be relatively simple to implement if you wanted to have a go?

**Comment by beyondmeat** [comment]
@ncw I took a stab. I know I'm close on the vfs.go edit but probably need some help.

**Comment by xd003** [comment]
Thanks for adding this, finally can get rid of rc.
Just wanted to confirm if the implementation is async ? I don't want the mount to be blocked until the dir tree read operation is complete

**Comment by ncw** [comment]
> Thanks for adding this, finally can get rid of rc. Just wanted to confirm if the implementation is async ? I don't want the mount to be blocked until the dir tree read operation is complete

Yes it is async - I added a note to the help to say it runs in the background.
